index.js

```diff
@@ -0,0 +1,22 @@
+var Cli = require("matrix-appservice-bridge").Cli;
+var AppServiceRegistration = require("matrix-appservice-bridge").AppServiceRegistration;
+
+var MatrixGitterBridge = require("./lib/MatrixGitterBridge");
+
+new Cli({
+  registrationPath: "gitter-registration.yaml",
+  bridgeConfig: {
+    schema: "config/gitter-config-schema.yaml",
+  },
+  generateRegistration: function(reg, callback) {
+    reg.setHomeserverToken(AppServiceRegistration.generateToken());
```

Comment:
You probably also want `reg.setId(AppServiceRegistration.generateToken());` or something, else the HS will whine at you that there is no `id` key.